services/ui-src/src/components/accordions/InstructionsAccordion.tsx

```diff
@@ -0,0 +1,72 @@
+import { Accordion, Box, ListItem, UnorderedList } from "@chakra-ui/react";
+import { AccordionItem } from "components";
+import { AnyObject } from "types";
+import { parseCustomHtml, sanitizeAndParseHtml } from "utils";
+
+export const InstructionsAccordion = ({ verbiage, ...props }: Props) => {
+  const { buttonLabel, intro, list, text } = verbiage;
+  return (
+    <Accordion allowToggle={true} allowMultiple={true} sx={sx.root} {...props}>
+      <AccordionItem label={buttonLabel} sx={sx.item}>
+        <Box sx={sx.textBox}>{parseCustomHtml(intro)}</Box>
+        {list && parseList(list)}
+        {text && <Box sx={sx.textBox}>{parseCustomHtml(text)}</Box>}
+      </AccordionItem>
+    </Accordion>
+  );
+};
+
+export const parseList = (list: any) => {
+  return (
+    <UnorderedList sx={sx.list}>
+      {list?.map((listItem: string | AnyObject, index: number) =>
+        typeof listItem === "string" ? (
+          <ListItem key={index}>{sanitizeAndParseHtml(listItem)}</ListItem>
+        ) : (
+          parseList(listItem)
+        )
+      )}
+    </UnorderedList>
+  );
+};
+
+interface Props {
+  verbiage: AnyObject;
+  [key: string]: any;
+}
```

Comment:
Thought: I'd love to see something more specific here.
* We know that the verbiage can only have 4 properties, 2 of which are required (buttonLabel, intro, list, text). So we could list those here to get better type safety than `AnyObject`.
* We know that there are a finite number of properties that are acceptable to pass through to Chakra's `Accordion`. We could restrict our props to only those by having `Props extends AccordionProps { ... }`
   * With AccordionProps being imported from Chakra
   * And with the `[key: string]: any` removed from our interface
   * We probably don't want the `allowToggle`, `allowMultiple`, or `sx` props overridden, so we could exclude those using `Omit` ( https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys )
* We know that, in all our other apps, the only prop we ever override on Chakra's Accordion is `defaultIndex={0}` - and we always need a comment explaining why. So perhaps we could skip the passthrough `...props` altogether, while removing the need for those explanations, by introducing a new named prop: `isOpenByDefault?: boolean`

Buuut this super-loose `Props` interface isn't actively harming us, either, so there's no need to address it as a part of this PR.